packages/crypto-aws-kms/src/utils.ts

```diff
@@ -94,6 +94,9 @@ export async function createKeyAlias({ alias, awsKeyId, kmsClient }: {
   );
 }
 
+/**
+ * Gets the `KeySpec` of the key specified.
```

Comment:
Should the models you're referring to using \` \` be linked in the docs?
Should the params be documented?

Reply:
Should return type be documented? is it auto documented?

Reply:
I'll take a stab if quick.

To be clear, the goal here is only to get rid of the annoying noise doing the minimum possible. Not interested in reading the code in detail to provide extensive and correct documentation.

Should params and return types be manually documented? Debatable, but I am not about to start that debate right now, I am following whoever created the rules for now, which obviously allows params and return types to be not documented.

Reply:
Addressed the comments.